lightningd/opening_control.c

```diff
@@ -281,6 +281,8 @@ static void funding_started_success(struct funding_channel *fc)
 				    fc->funding_scriptpubkey);
 		if (fc->our_upfront_shutdown_script)
 			json_add_hex_talarr(response, "close_to", fc->our_upfront_shutdown_script);
+		json_add_string(response, "warning",
```

Comment:
Just that rename this field to warning_usage?

All warning fields in our JSON have specific sematics (this is useful for translations, for example).

You'll also need to add it to the JSON schema in doc/schemas/fundchannel.schema.json

Reply:
Renamed to `warning_usage`. In `doc/schemas/fundchannel.schema.json` I'm only seeing input parameters, should I just add this output parameter to it?